### ISSUE TITLE
What new features were added to this HEN?

### ISSUE THREAD

**Issue by RobbedColek** [Karf5/Playstation-4-Special-Homebrew-Enabler#1]
I am asking, because the only difference between your repository and official HEN-VTX is addition of your nickname to the pop-up screen - as can be verified when diffing the repositories.

![Screenshot_1](https://github.com/user-attachments/assets/cf13f143-e527-4841-858e-bc9209dc36e7)
![Clipboard01](https://github.com/user-attachments/assets/6d64e6bd-a50e-4e98-804b-c1c42c595ab7)

Also your repository is missing submodules and has build files which shouldn't be there if uploaded to Git - perhaps educate yourself a little more how Git works as well 🤡.

**Comment by Tribb** [comment]
yes stolen code from stooged too with his tvbox crap, What an idiot, Proper Devs dont upload others code, they clone and PR

**Comment by Karf5** [comment]
Firstly I didn't steal anything. Secondly, you seem to be blind because I have added everything about ( Playstation-4-Special-Homebrew-Enabler ) in the README file and I have mentioned that I have added the feature of blocking system updates. I don't know why
This is strange to you. Didn't SiSTR0 take the source code of HEN-VTX and develop it, update it, add more features to it, rename it, put his name on it, and make the project closed source? Therefore, I decided to improve ( Playstation-4-Special-Homebrew-Enabler ) and add more features to it, and I will continue to support it. But again it seems that I am talking to blind and stupid people 🤣

**Comment by Karf5** [comment]
Firstly I didn't steal anything. Secondly, you seem to be blind because I have added everything about ( Playstation-4-Special-Homebrew-Enabler ) in the README file and I have mentioned that I have added the feature of blocking system updates. 

I don't know why
This is strange to you. Didn't SiSTR0 take the source code of HEN-VTX and develop it, update it, add more features to it, rename it, put his name on it, and make the project closed source? Therefore, I decided to improve ( Playstation-4-Special-Homebrew-Enabler ) and add more features to it, and I will continue to support it.

 But again it seems that I am talking to blind and stupid people 🤣
